javascripts/login.js

```diff
@@ -0,0 +1,92 @@
+function logIn() {
+    var email = $("#email").val();
+    var pwd = $("#pwd").val();
+    if (checkEmail(email) && checkPassword(pwd)) {
+        var expdate = new Date();
+        expdate.setTime(expdate.getTime() + (365*24*60*60*1000));
+        document.cookie = "username=RandomUserName; expires=" + expdate.toUTCString() + "; path=/";
+        document.location = "index.html";
+        return true;    
+    } else { 
+        return false;
+    }   
+}   
+
+function signUp() {
+    var usr1 = new User();
+    if(usr1.valid()) {
+        alert("Success!");
+        return true;  // ...new User properties could be saved 
+    } else {
+        return false; // ...new User properties could be erased
+    }
+}
+
+function User() {
+    this.firstname = $("#reg-firstname").val();
+    this.lastname = $("#reg-lastname").val();
+    this.email = $("#reg-email").val();
+    this.birthday = $("#reg-birthday").val();
+    this.password = $("#reg-pwd").val();
+    this.valid = function regValidation() { 
+        if (checkName(this.firstname, this.lastname) && checkEmail(this.email) && checkBirthday(this.birthday) && checkPassword(this.password) && checkPasswordConf(this.password, $("#reg-pwd-conf").val())) {
+            return true;
```

Comment:
Please be patient in logic decisions,))
this.valid = function regValidation() { 
   return (checkName(this.firstname, this.lastname) && checkEmail(this.email) && checkBirthday(this.birthday) && checkPassword(this.password) && checkPasswordConf(this.password, $("#reg-pwd-conf").val()))
   }